python/run-tests

```diff
@@ -18,165 +18,9 @@
 #
 
 
-# Figure out where the Spark framework is installed
-FWDIR="$(cd "`dirname "$0"`"; cd ../; pwd)"
+FWDIR="$(cd "`dirname $0`"/..; pwd)"
+cd "$FWDIR"
 
-. "$FWDIR"/bin/load-spark-env.sh
+exec python -u ./python/run-tests.py
 
-# CD into the python directory to find things on the right path
-cd "$FWDIR/python"
-
-FAILED=0
-LOG_FILE=unit-tests.log
-START=$(date +"%s")
-
-rm -f $LOG_FILE
-
-# Remove the metastore and warehouse directory created by the HiveContext tests in Spark SQL
-rm -rf metastore warehouse
```

Comment:
@ahirreddy, I think that you originally added these lines.  Do we still need them?  AFAIK the pull request builder's `git clean -fdx` should take care of this for us, but was there another reason why we need this?